### ISSUE TITLE
soumith/pty.js does not compile on arch linux

### ISSUE THREAD

**Issue by bobbens** [clementfarabet/gfx.js#42]
soumith/pty.js does not compile on arch linux. Solution is to change it to chjj/pty.js in the package.json and it does work.

**Comment by soumith** [comment]
You should just move to szym/display , we've abandoned gfx.js due to all the build issues.

**Comment by bobbens** [comment]
Yes. I don't intend to use it for new code. I just wanted to try some of the t-sne demos and such which are still all written to use gfx.js.
